docs/EXTENDED-ARCHIVES.md

```diff
@@ -0,0 +1,344 @@
+
+_Some examples of devices are: Beaglebone Black, Beaglebone Green, Intel Edison,
+and Raspberry Pi 3._
+
+Here's an example of a real-world device manifest for Raspberry Pi 3:
+
+```json
+{
+  "displayName": "Raspberry Pi 3",
+  "arch": "armv7hf",
+  "slug": "raspberrypi3",
+  "slugAliases": [
+    "raspberry-pi3",
+    "rpi3"
+  ],
+  "url": "https://www.raspberrypi.org/products/raspberry-pi-3-model-b/",
+  "photoPath": "raspberry-pi-3-photo.jpg"
+}
+```
+
+A device may be described with any of the following properties:
+
+### `displayName (String)`
+
+The human-friendly name of the device.
+
+### `arch (String[])`
+
+The device architecture.
+
+The current possible values are: `aarch64`, `amd64`, `armv5e`, `armv7hf`,
+`i386`, and `rpi`.
+
+### `slug (String)`
+
+The device slug.
+
+### `slugAliases (String[])`
+
+The device slug aliases, if any.
+
+### `url (String)`
+
+The url to the landing page of the device.
+
+### `photoPath (String)`:
+
+The path to a photo of the device in JPEG format.
+
+### `compatibleWith (String[])`
+
+An array of the device slugs this device is compatible with.
+
+Declaring compatibility allows images that target a specific device to be also
+used in other device types that are known to work with it.
+
+Image
+-----
+
+> An image represents a downloadable operating system image that can be booted
+  in a device or family of devices.
+
+_Some examples of images are: Raspbian Jessie, ResinOS, CoreOS, and Ostro._
+
+Here's an example of a real-world image manifest for Raspbian Jessie:
+
+```json
+{
+  "displayName": "Raspbian Jessie",
+  "version": "May 2016",
+  "url": "https://www.raspbian.org",
+  "releaseNotesUrl": "http://downloads.raspberrypi.org/raspbian/release_notes.txt",
+  "logoPath": "image-logo.svg",
+  "checksumType": "sha1",
+  "checksum": "64c7ed611929ea5178fbb69b5a5f29cc9cc7c157",
+  "recommendedDriveSize": 4294967296,
+  "bmapPath": "raspbian-jessie.bmap",
+  "instructionsPath": "raspbian-getting-started.markdown",
+  "updateUrl": "https://downloads.raspberrypi.org/raspbian_latest",
+  "etag": "c0170-53152af2-533d18ef29fc0",
+  "supportedDevices": [
+    "raspberrypi",
+    "raspberrypi2",
+    "raspberrypi3"
+  ]
+}
+```
+
+A image may be described with any of the following properties:
+
+### `displayName (String)`
+
+The human-friendly name of the image.
+
+### `version (String)`
+
+The version of the image.
+
+### `url (String)`
+
+The main url of the image.
+
+### `releaseNotesUrl (String)`
+
+The url to the image's release notes or CHANGELOG.
+
+### `logoPath (String)`
+
+The path to the image specific logo in SVG format.
+
+The SVG format requirement ensures that the logo looks good in all types of
+screen resolutions.
+
+### `checksumType (String)`
+
+The checksum type. The current possible values are: `sha1`, `crc32`, and `md5`.
+
+### `checksum (String)`
+
+The actual checksum according to the type declared in `checksumType`.
+
```

Comment:
Yeah, I guess it would be the 'fastest one that's still effective checking against for corruption', which might vary per machine given for instance [hardware acceleration of SHA](https://en.wikipedia.org/wiki/Intel_SHA_extensions).
But also, another image writing tool could want to make use of the metadata and verify the image but doesn't support all the functions that etcher does. Theoretically. Just trying to think of the future. This would probably be one of the main pieces of metadata another program might want to leverage.